crates/sncast/tests/e2e/account/create.rs

```diff
@@ -337,33 +329,52 @@ pub async fn test_keystore_without_account() {
         DEVNET_OZ_CLASS_HASH,
     ];
 
-    let snapbox = runner(&args);
+    let snapbox = runner(&args).current_dir(temp_dir.path());
 
     snapbox.assert().stderr_matches(indoc! {r"
         command: account create
         error: Argument `--account` must be passed and be a path when using `--keystore`
     "});
-
-    _ = fs::remove_file(keystore_path);
 }
 
-#[test_case("tests/data/keystore/my_key.json", "tests/data/keystore/my_account_new.json", "error: Keystore file tests/data/keystore/my_key.json already exists" ; "when keystore exists")]
-#[test_case("tests/data/keystore/my_key_new.json", "tests/data/keystore/my_account.json", "error: Account file tests/data/keystore/my_account.json already exists" ; "when account exists")]
+#[test_case("./tests/data/keystore/my_key.json", "./tests/data/keystore/my_account_new.json", "error: Keystore file my_key.json already exists" ; "when keystore exists")]
+#[test_case("./tests/data/keystore/my_key_new.json", "./tests/data/keystore/my_account.json", "error: Account file my_account.json already exists" ; "when account exists")]
 pub fn test_keystore_already_exists(keystore_path: &str, account_path: &str, error: &str) {
+    let temp_dir = tempdir().expect("Unable to create a temporary directory");
+
+    let keystore_path = Path::new(keystore_path);
+    if keystore_path.exists() {
```

Comment:
why do we need to check if this exists?

Reply:
To copy only those files that exist, as the test cases are the following:
1. Keystore file already exists; Account files does not exist
2. Keystore file does not exist; Account file already exists

Reply:
Let's split this test into two then, to keep test cases as simple as possible (withouth ifs etc)